lettuce/collision.py

```diff
@@ -3,20 +3,38 @@
 """
 
 from lettuce.equilibrium import QuadraticEquilibrium
+import torch
 
 
 class BGKCollision:
     def __init__(self, lattice, tau):
         self.lattice = lattice
         self.tau = tau
-
     def __call__(self, f):
         rho = self.lattice.rho(f)
         u = self.lattice.u(f)
         feq = self.lattice.equilibrium(rho, u)
         f = f - 1.0/self.tau * (f-feq)
         return f
 
+class BGKCollision_guo:
```

Comment:
This is really a `BGKCollisionWithForce` or something like this. You can specify the forcing scheme as a command line argument.